hub/integrations/tests/test_pytorch.py

```diff
@@ -23,7 +28,7 @@ def test_pytorch_small(ds):
         ds.create_tensor("image2")
         ds.image2.extend(np.array([i * np.ones((100, 100)) for i in range(256)]))
 
-    if isinstance(remove_memory_cache(ds.storage), MemoryProvider):
+    if PY38 and isinstance(remove_memory_cache(ds.storage), MemoryProvider):
```

Comment:
Good catch. We should probably run our tests on both 3.6 and 3.8 to prevent such slipups in the future.